integration/kubernetes/k8s-cpu-ns.bats

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+
+setup() {
+	export KUBECONFIG=/etc/kubernetes/admin.conf
+	pod_name="constraints-cpu-test"
+	cpu_limit=1
+	cpu_requests="500m"
+}
+
+@test "Check CPU constraints" {
+	wait_time=30
+	sleep_time=5
+
+	# Create the pod
+	sudo -E kubectl create -f pod-cpu.yaml
+
+	# Check pod creation
+	pod_status_cmd="sudo -E kubectl get pods -a | grep $pod_name | grep Running"
+	waitForProcess "$wait_time" "$sleep_time" "$pod_status_cmd"
+
+	# Check cpu limits
+	cpu_limits_information=$(sudo -E kubectl describe pods $pod_name | grep -A 1 "Limits" | grep cpu | cut -d ':' -f2)
```

Comment:
I think that in addition, we should check inside the container that the cpu limits are correct. 
wdyt @devimc

Reply:
yes we should check the constraint inside the container